content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -5,16 +5,16 @@ weight: 70
 aliases:
     - /docs/setup/kubernetes/install/cni
     - /docs/setup/kubernetes/additional-setup/cni
-keywords: [kubernetes,cni,sidecar,proxy,network,helm]
-owner: istio/wg-environments-maintainers
-test: no
+keywords: [cni]
+owner: istio/wg-networking-maintainers
+test: yes
 ---
 
 Follow this guide to install, configure, and use an Istio mesh using the Istio Container Network Interface
 ([CNI](https://github.com/containernetworking/cni#cni---the-container-network-interface))
 plugin.
 
-By default Istio injects an `initContainer`, `istio-init`, in pods deployed in
+By default Istio injects an init container, `istio-init`, in pods deployed in
```

Comment:
this is inconsistent throughout?  I think it's in quotes to bypass spellchecking.

Reply:
Updated all presence of `initContainer` to init container.